samples/client/petstore/typescript-fetch/builds/default/apis/PetApi.ts

```diff
@@ -23,7 +23,7 @@ import {
 } from '../models';
 
 export interface AddPetRequest {
-    body: Pet;
+    pet: Pet;
```

Comment:
Look like you're using an old version. please pull the latest master and merge into your branch and then update the samples again.